build-trusses/build-x86_64.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN sudo apk update &&\
     sudo chown root:root /etc/sudoers.d/docker &&\
     sudo sed -i.bak 's/^Defaults.*requiretty//g' /etc/sudoers
 
-USER docker
+#USER docker

```

Comment:
You may want to add a comment that this is commented out currently due to UID conflicts with alpine distros

Reply:
added comment